### PR TITLE
Bump pg-mon commit

### PR DESCRIPTION
**Pull request by RafiaSabih**
Update the commit-id and install after init.

### CODE REVIEW

File: postgres-appliance/scripts/configure_spilo.py [hunk 1]
```diff
@@ -256,7 +256,7 @@ def deep_update(a, b):
     {{/SSL_CRL_FILE}}
     ssl_cert_file: {{SSL_CERTIFICATE_FILE}}
     ssl_key_file: {{SSL_PRIVATE_KEY_FILE}}
-    shared_preload_libraries: 'bg_mon,pg_stat_statements,pgextwlist,pg_auth_mon,set_user'
+    shared_preload_libraries: 'bg_mon,pg_stat_statements,pgextwlist,pg_auth_mon,set_user,pg_mon'
```
Comment (CyberDem0n) [inline postgres-appliance/scripts/configure_spilo.py:259]: It should be done conditionally, i.e. [here](https://github.com/zalando/spilo/pull/456/files#diff-5e29afdf8025daafbc08cc254eb83f9fR35-R40) and [here](https://github.com/zalando/spilo/blob/feature/pg13/postgres-appliance/bootstrap/pg_upgrade.py#L20-L25). Preferably only for version 13.

File: postgres-appliance/scripts/post_init.sh [hunk 2]
```diff
@@ -36,6 +36,10 @@ CREATE EXTENSION IF NOT EXISTS pg_auth_mon SCHEMA public;
 ALTER EXTENSION pg_auth_mon UPDATE;
 GRANT SELECT ON TABLE public.pg_auth_mon TO robot_zmon;
 
+CREATE EXTENSION IF NOT EXISTS pg_mon SCHEMA public;
+ALTER EXTENSION pg_mon UPDATE;
+GRANT SELECT ON TABLE public.pg_mon TO robot_zmon;
```
Comment (CyberDem0n) [inline postgres-appliance/scripts/post_init.sh:41]: I think public already has SELECT permissions.

File: postgres-appliance/scripts/post_init.sh [hunk 3]
```diff
@@ -36,6 +36,10 @@ CREATE EXTENSION IF NOT EXISTS pg_auth_mon SCHEMA public;
 ALTER EXTENSION pg_auth_mon UPDATE;
 GRANT SELECT ON TABLE public.pg_auth_mon TO robot_zmon;
 
+CREATE EXTENSION IF NOT EXISTS pg_mon SCHEMA public;
+ALTER EXTENSION pg_mon UPDATE;
```
Comment (CyberDem0n) [inline postgres-appliance/scripts/post_init.sh:40]: It needs to be done somewhere [here](https://github.com/zalando/spilo/pull/456/files#diff-ab31b078396f155439631156cf699b44R156-R157).

File: postgres-appliance/scripts/configure_spilo.py [hunk 4]
```diff
@@ -36,7 +36,8 @@
     'timescaledb':    (9.6, 12, True,  True),
     'pg_cron':        (9.5, 13, True,  False),
     'pg_stat_kcache': (9.4, 13, True,  False),
-    'pg_partman':     (9.4, 13, False, True)
+    'pg_partman':     (9.4, 13, False, True),
+    'pg_mon':         (10, 13, True, True)
```
Comment (CyberDem0n) [inline postgres-appliance/scripts/configure_spilo.py:40]: (10, 13, True, **False**)

**Comment by RafiaSabih** [comment]
:+1:

**Comment by Jan-M** [comment]
:+1:

**Comment by CyberDem0n** [comment]
:+1:

**Pull request merged**